dev_guide/projects.adoc

```diff
@@ -104,7 +104,7 @@ The right panel shown with the service catalog provides quick access to the most
 recently accessed projects (up to five projects). For the full list of projects,
 use the *View All* link provided at the top of the right panel.
 
-image::3.7-project-list.png[Projects List]
+image::3.9-project-list.jpg[Projects List]
```

Comment:
This looks like the overview instead of project list (if I'm looking at the right image)

Reply:
Here's the 3.9 project list 
https://drive.google.com/file/d/1nexOHsTgJ7QorJBq69EzW4ZEhm1gYWzy/view?usp=sharing

Reply:
Whoops! Fixed. Thanks!